core/src/Streamly/Internal/Data/Stream/StreamD/Exception.hs

```diff
@@ -8,16 +8,8 @@
 
 module Streamly.Internal.Data.Stream.StreamD.Exception
     (
-      gbracket_
```

Comment:
Do not remove anything from this module, these just have to be adapted to IO monad in the next PR.